configuration/redis/metadata.yaml

```
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: configuration
+name: redis
+version: v1
+status: alpha
+title: "Redis"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-configuration-stores/redis-configuration-store/
+capabilities: []
+metadata:
+  - name: redisHost
+    required: true
+    description: Connection-string for the redis host
+    example: "redis-master.default.svc.cluster.local:6379"
+    type: string
+  - name: redisPassword
+    required: false
+    description: Password for Redis host. No Default. Can be secretKeyRef to use a secret reference
+    example:  "KeFg23!"
+    type: string
+  - name: enableTLS
+    required: false
+    description: | 
+      If the Redis instance supports TLS with public certificates, can be
+      configured to be enabled or disabled. Defaults to false.
+    example: "false"
+    default: "false"
+    type: bool
+  - name: maxRetries
+    required: false
+    description: |
+      Maximum number of retries before giving up. Defaults to "3".
+    default: "3"
+    example: "5"
+    type: number
+  - name: maxRetryBackoff
+    required: false
+    description: |
+      Maximum backoff between each retry in milliseconds. Defaults to 2 seconds; "-1" disables backoff.
```

We really need to change these durations, nanoseconds are ridiculous...

```suggestion
      Maximum backoff between each retry in nanoseconds. Defaults to 2 seconds; "-1" disables backoff.
```

---

I know :) And it is not even uniform how we handle those among the many Redis components we have.